packages/core/src/app/customer/CheckoutButtonContainer.spec.tsx

```diff
@@ -22,6 +22,7 @@ describe('CheckoutButtonContainer', () => {
         localeContext = createLocaleContext(getStoreConfig());
 
         jest.spyOn(checkoutState.data, 'getCustomer').mockReturnValue(getGuestCustomer());
+        jest.spyOn(checkoutState.data, 'isPaymentDataRequired').mockReturnValue(true);
```

Comment:
should you cover the case in test when `isPaymentDataRequired` returns false?

Reply:
Done